python/lsst/ts/m2/mock_command.py

```diff
@@ -23,72 +23,163 @@
 
 from lsst.ts.idl.enums import MTM2
 
-from . import DetailedState
+from . import DetailedState, CommandStatus
 
 
 __all__ = ["MockCommand"]
 
 
 class MockCommand:
-    """Mock command to simulate the execution of command in real hardware.
+    """Mock command to simulate the execution of command in real hardware."""
 
-    TODO (DM-30851):
-    Most of code will be implemented in DM-30851.
-
-    Attributes
-    ----------
-    system_enabled : `bool`
-        System is in the Enabled state or not.
-    """
-
-    def __init__(self):
-
-        # System is in the Enabled state or not
-        # TODO: This attribute will be removed in the work of DM-30851
-        self.system_enabled = False
-
-    async def enable(self, message, message_event):
+    async def enable(self, message, model, message_event):
```

Comment:
I see that here and in other methods you are passing the `model` class to `MockCommand`. Consider making it a class member so you don't have to do it, e.g.;

You could do:

```
def __init__(self, model):

    self.model = model
```

and then use `self.model` everywhere else.

Reply:
I thought in that way as well. However, I gave up because I thought it might be better to only let the `MockServer` to hold the `MockModel` as an attribute. I also intentionally to express this in:

```
self.model, command_status = await self._command_response[command_name](
                message, self.model, self._message_event
)
```